datamarket/constants.ts

```diff
@@ -2,6 +2,12 @@
 // INTERFACES/TYPES //
 //////////////////////
 
+import internal from "stream";
+
+export type SortType = "Price ($-$$$)" | "Price ($$$-$)" | "Name (A-Z)" | "Name (Z-A)";
+export type ViewType = "Gallery" | "List";
```

Comment:
this is fine, but an enum is usually better.
ie
`export enum ViewType {GALLERY = "Gallery"}` (or something, u can ask chatgpt)
then you use it like
ViewType.GALLERY

Reply:
Done